lib/cdo/help_header.rb

```diff
@@ -59,10 +59,9 @@ def self.get_help_contents(options)
     end
 
     if options[:level] && options[:level].game == Game.music
-      options[:request].path.starts_with?('/projectbeats')
       entries << {
-        title: I18n.t("#{loc_prefix}project_beats_documentation"),
-        url: "https://studio.code.org/docs/ide/projectbeats/",
+        title: I18n.t("#{loc_prefix}music_lab_documentation"),
+        url: "https://studio.code.org/docs/ide/music/",
```

Comment:
So long as opening in a new tab still works, it might be good to use a relative URL here so that it stays in the same environment.

Reply:
It does open in a new tab.

so that'd be `CDO.code_org_url('/docs/ide/music/')` I believe. I wonder if I should change them all... and why specifically the doc links are all pointing directly to prod.

Reply:
Since we're already on dashboard, I'm not sure it even needs to be wrapped in `CDO.code_org_url()`?

Reply:
You are correct.